tests/e2e/util.go

```diff
@@ -5635,3 +5635,34 @@ func startCSIPods(ctx context.Context, client clientset.Interface, csiReplicas i
 	isServiceStopped := false
 	return isServiceStopped, err
 }
+
+// enableFullSyncFss enables full sync fss in internal features configmap in csi namespace
+func enableFullSyncTriggerFss(ctx context.Context, client clientset.Interface, namespace string, fss string) {
+	fssCM, err := client.CoreV1().ConfigMaps(namespace).Get(ctx, csiFssCM, metav1.GetOptions{})
+	framework.Logf("%s configmap in namespace %s is %s", csiFssCM, namespace, fssCM)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	fssFound := false
+	for k, v := range fssCM.Data {
+		if fss == k && v != "true" {
+			framework.Logf("FSS %s found in the %s configmap in namespace %s", fss, csiFssCM, namespace)
+			fssFound = true
+			fssCM.Data[fss] = "true"
+			_, err = client.CoreV1().ConfigMaps(namespace).Update(ctx, fssCM, metav1.UpdateOptions{})
+			framework.Logf("%s configmap in namespace %s is %s", csiFssCM, namespace, fssCM)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			csipods, err := client.CoreV1().Pods(csiSystemNamespace).List(ctx, metav1.ListOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			for _, pod := range csipods.Items {
```

Comment:
dump csi logs before killing them, ref `collectPodLogs()`
also add a comment here mentioning why we are doing this

Reply:
Done